notNeededPackages.json

```diff
@@ -712,6 +712,10 @@
             "libraryName": "cosmiconfig",
             "asOfVersion": "6.0.0"
         },
+        "countries-and-timezones": {
+            "libraryName": "countries-and-timezones",
+            "asOfVersion": "3.2.2"
```

Comment:
```suggestion
            "asOfVersion": "3.2.0"
```

Reply:
That probably makes sense, although I think I need to clarify something first, since I'm not a regular contributor here. The types added in v3.2.0 wouldn't compile if you used `skipLibCheck: false` in your tsconfig. v3.2.2 is the first version where the types work correctly. Is it better to make the change you suggested to this PR, or leave it as it is?

Reply:
Oh, ok. Make sense, thx!